docs/querying/sql-functions.md

```diff
@@ -1756,39 +1809,66 @@ Returns the following:
 
 ## DECODE_BASE64_COMPLEX
 
-Decodes a Base64-encoded string into a complex data type, where `dataType` is the complex data type and `expr` is the Base64-encoded string to decode.
+Decodes a Base64-encoded expression into a complex data type.
+
+You can use the function to ingest data when a column contains an encoded data sketch such as Theta or HLL.
 
-* **Syntax**: `DECODE_BASE64_COMPLEX(dataType, expr)`
+The function supports `hyperUnique` and `serializablePairLongString` data types by default.
+You can enable support for the following complex data types by [loading their extensions](../configuration/extensions.md):
+
+- `druid-bloom-filter`: `bloom`
+- `druid-datasketches`: `arrayOfDoublesSketch`, `HLLSketch`, `KllDoublesSketch`, `KllFloatsSketch`, `quantilesDoublesSketch`, `thetaSketch`
+- `druid-histogram`: `approximateHistogram`, `fixedBucketsHistogram`
+- `druid-stats`: `variance`
+- `druid-compressed-bigdecimal`: `compressedBigDecimal`
+- `druid-momentsketch`: `momentSketch`
+- `druid-tdigestsketch`: `tDigestSketch`
+
+* **Syntax:** `DECODE_BASE64_COMPLEX(dataType, expr)`
 * **Function type:** Scalar, other
 
-[Learn more](sql-scalar.md#other-scalar-functions)
+<details><summary>Example</summary>
```

Comment:
These examples are a little different than the rest of the reference because they refer to an imaginary value `theta_input`. My suggestion is to remove the second example and add the following results for the first:

```sql
DECODE_BASE64_COMPLEX('thetaSketch', "theta_input") AS "theta_decoded"
```

Returns the following:

|`theta_decoded`|
|---|
|`AgMDAAAazJNBAAAAAACAP...`|